rules/missing_punctuation.yml

```diff
@@ -0,0 +1,14 @@
+id: missing_punctuation
```

Comment:
I think it would be hard to enforce the grouped structure of our imports with a rule in general but maybe we could add a rule to make sure `use super::*;` is always first in the test imports and always followed by an empty line. Quickly looking through the code it seems we are a little inconsistent here.

Reply:
Added to task.